lib/command/run.rb

```diff
@@ -442,16 +442,22 @@ def print_detached_commands
       )
     end
 
-    def resolve_job_status
-      result = cp.fetch_cron_workload(runner_workload, location: location)
-      job_details = result&.dig("items")&.find { |item| item["id"] == job }
-      status = job_details&.dig("status")
+    def resolve_job_status # rubocop:disable Metrics/MethodLength
+      loop do
```

Comment:
@dzirtusss should we have some catch if this loop is going to be infinite, like the workload got renamed so `result` is always nil?

Reply:
https://github.com/shakacode/control-plane-flow/blob/fe6493f86956b1e2e89ff140abeb96b5391faec9/lib/core/controlplane.rb#L245C1-L248C6

```ruby
  def fetch_cron_workload(workload, location:)
    cmd = "cpln workload cron get #{workload} #{gvc_org} --location #{location} -o yaml"
    perform_yaml(cmd)
  end
```

`perform_yaml` should always add the `-o yaml`